### PR TITLE
Boilerplate for bokeh/client

### LINKED ISSUE #8207: Adding/Updating Boilerplate Code
As part of the NumFocus small grant, we are funded to "Update all Bokeh source files with common boilerplate and header format (started but not yet finished)". The work should cover the following directories:

- [x] Top Level (#8220)
- [x] _testing
- [x] application (#8245)
- [x] client (#8276)
- [x] colors (#8293)
- [x] command (#8313)
- [x] core (#8341)
- [x] document
- [x] embed
- [x] io
- [x] models
- [x] plotting
- [x] protocol
- [x] sampledata
- [x] server
- [x] sphinxext
- [x] tests
- [x] themes
- [x] util

Non-directory:
- [x] Copyright
- [x] Update __all__ for transitive imports

Note that test module boilerplate varies from the standard.

### PR DESCRIPTION
**Pull request by HuntJSparra**
- [x] issues: addresses part of #8207 
- [x] tests added / passed

Adding/updating boilerplate in files under bokeh/client directory

### CODE REVIEW

File: bokeh/client/tests/test_states.py [hunk 1]
```diff
@@ -35,6 +34,8 @@
 class MockConnection(object):
     def __init__(self, to_pop=None): self._to_pop = to_pop
 
+    # Private methods ---------------------------------------------------------
+
```
[the remark this answers is not in the document]
Comment (HuntJSparra) [reply bokeh/client/tests/test_states.py]: This makes sense. Would you like me to go back and remove organization comments from the toplevel and bokeh/application tests?

File: bokeh/client/states.py [hunk 2]
```diff
@@ -45,6 +53,9 @@ class NOT_YET_CONNECTED(object):
     ''' The ``ClientConnection`` is not yet connected.
 
     '''
+
+    # Public methods ----------------------------------------------------------
+
```
[the remark this answers is not in the document]
Comment (HuntJSparra) [reply bokeh/client/states.py]: Do you want added comments removed from the toplevel files and from bokeh/application's files?

File: bokeh/client/tests/test_states.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bryevdv) [reply bokeh/client/tests/test_states.py]: No, let's just do it this way moving forward

File: bokeh/client/states.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (bryevdv) [reply bokeh/client/states.py]: No that's fine, let's just skip them here and moving forward.

File: bokeh/client/websocket.py [hunk 3]
```diff
@@ -53,7 +57,7 @@ def __init__(self, socket):
         # messages atomically.
         self.write_lock = locks.Lock()
 
-    # Internal methods --------------------------------------------------------
```
Comment (HuntJSparra) [inline bokeh/client/websocket.py]: Would you rather call these "Internal methods" to be more pythonic or "Private methods"?

Comment (bryevdv) [reply bokeh/client/websocket.py]: I don't think we should try to organize/label method groups unsure classes with comments at all at this point, it's just a big task to get bogged down in, and also there has just not been any real thought about what groupings would be useful or make sense

**Comment by HuntJSparra** [comment]
Ping @bryevdv

**Pull request merged**